packages/pyroscope-models/src/flamebearer.ts

```diff
@@ -40,7 +40,8 @@ export type Flamebearer = {
 
 export type addTicks =
   | { format: 'double'; leftTicks: number; rightTicks: number }
-  | { format: 'single' };
+  // todoL: remove left/right ticks
+  | { format: 'single'; leftTicks?: number; rightTicks?: number };
```

Comment:
What's going on here? My understanding is that `single` doesn't have `{left,right}Ticks`